src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -214,7 +214,7 @@ private function addCsrfSection(ArrayNodeDefinition $rootNode): void
                     ->addDefaultsIfNotSet()
                     ->children()
                         // defaults to framework.session.enabled && !class_exists(FullStack::class) && interface_exists(CsrfTokenManagerInterface::class)
-                        ->booleanNode('enabled')->defaultNull()->end()
+                        ->scalarNode('enabled')->defaultNull()->end()
```

Comment:
bugfix: booleanNode cannot default to null
still part of this PR because I'm not sure we should backport this to earlier branches - it might be a needles behavior change, WDYT?

Reply:
are you saying `booleanNode()->defaultsNull()` is broken everywhere?

Reply:
Yes that's whay I mean, it's not supported. PR welcome if we ant to improve ;)